kge/model/embedder/lookup_embedder.py

```diff
@@ -77,6 +78,48 @@ def normalize_embeddings(job):
 
             job.pre_batch_hooks.append(normalize_embeddings)
 
+    @torch.no_grad()
+    def init_pretrained(
+        self,
+        pretrained_embedder: torch.nn.Embedding,
+        ids_pretrained_model: List,
+        ids_new_model: List,
+        ensure_all: bool = False,
+    ) -> None:
+        """
+        Initialize embedding layer with pre-trained embeddings from another embedding
+        layer.
+        Maps embeddings based on the entity/relation ids provided
+        Args:
+            pretrained_embedder: Embedding layer with pre-trained embeddings
+            ids_pretrained_model: entity/relation ids corresponding to the embeddings
+                                  of the pre-trained embedding layer
+            ids_new_model: entity/relation ids corresponding to the embeddings of the
+                           model to be initialized
+            ensure_all: ensure that all entities/relations can be initialized with
+                        embeddings provided in the pre-trained model
+
+        Returns:
+            None
+        """
+        _, dataset_intersect_ind, checkpoint_intersect_ind = np.intersect1d(
```

Comment:
I suggest to use ocnsistent naming (e.g., self_intersect_ind, pretrained_intersect_ind)

Reply:
It may be more efficient to intersect the id indexes directly.

Reply:
I'm not sure what you mean by intersecting on the id indexes directly

Reply:
I mean dataset.entity_ids(), But I thought we also have inverted indexes, but apparently we don't ;). Won't matter too much anyway, since this is a one time cost.